source/windows/thread.c

```diff
@@ -0,0 +1,93 @@
+/*
+* Copyright 2010 - 2018 Amazon.com, Inc. or its affiliates.All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file.This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied.See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+#include <aws/common/thread.h>
+
+struct thread_wrapper {
+    struct aws_allocator *allocator;
+    void(*func)(void *arg);
+    void *arg;
+};
+
+static DWORD thread_wrapper_fn(LPVOID arg) {
```

Comment:
This needs to be `static DWORD WINAPI`. The `WINAPI` modifies the calling convention for the function.

Reply:
good catch.